Add Licensing section (#5624)
DevExpress/devextreme-documentation: concepts/Common/Licensing/20 Sharing Keys by Multiple Developers.md

According to the EULA, every developer who works with DevExtreme must have a license, since the DevExtreme license has a per seat basis.

> If you have many licenses that you distribute among developers, then each developer will have access to their own personal key in Download Manager under their account.

Reviewer: ```suggestion
If you have many licenses you can assign them to different developers, so that each developer will have their own personal key in Download Manager under their account.
```

Reviewer 2: under their DevExpress account

> If a developer leaves or simply stops working with DevExtreme, then you can revoke their license in your personal account. This will not affect the license keys. They will still continue to work within the major versions for which they were released.

Reviewer: ```suggestion
If a developer leaves or simply stops working with DevExtreme, then you can revoke their license in your personal account. However, this will not affect the license keys that was already issued. it will remain to be valid for all updates under the major version to which it was bound (and the older ones)
```
